src/main/java/ru/skypro/homework/controller/UsersController.java

```diff
@@ -0,0 +1,53 @@
+package ru.skypro.homework.controller;
+
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+import ru.skypro.homework.dto.CurrentUser;
+import ru.skypro.homework.dto.Login;
+import ru.skypro.homework.dto.NewPassword;
+import ru.skypro.homework.dto.Register;
+import ru.skypro.homework.dto.SetPassw;
+import ru.skypro.homework.service.AuthService;
+import ru.skypro.homework.service.UsersService;
+
+import java.security.Principal;
+
+
+@Slf4j
+@CrossOrigin(value = "http://localhost:3000")
+@RestController
+@RequiredArgsConstructor
+public class UsersController {
```

Comment:
```suggestion
public class UserController {
```
в единственном числе называем классы